devops/setup.android.env.sh

```diff
@@ -69,7 +69,7 @@ download_and_unzip_dependencies_for_all_architectures(){
     #TODO Get dependencies in more optimized way
     pushd ${ANDROID_BUILD_FOLDER}
         if [ ! -d "indy-android-dependencies" ] ; then
-            git clone https://github.com/evernym/indy-android-dependencies.git
+            git clone https://github.com/sovrin/indy-android-dependencies.git
             pushd ${ANDROID_BUILD_FOLDER}/indy-android-dependencies/prebuilt/
```

Comment:
https://github.com/sovrin/indy-android-dependencies is not a valid repo.  Should  be sovrin-foundation.

Reply:
This repo must be forked first by sovrin-foundation

Reply:
I think it should be moved to downloading of dependencies from repo.sovrin.org like in libindy

Reply:
@KitHat do we have a way to fix this? Have you added the fix?